drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -183,8 +183,10 @@
 .xblock--drag-and-drop .drag-container .option[draggable='true']:hover,
 .xblock--drag-and-drop .drag-container .option[draggable='true'][aria-grabbed='true'] {
     outline-width: 2px;
-    outline-style: solid;
+    outline-style: dotted;
     outline-offset: -4px;
+    /* IE 11 doesn't support outline-offset, so we use a box-shadow instead. */
+    box-shadow: inset 0 0 0 2px;
```

Comment:
Or maybe not... I'm having trouble finding what the individual arguments are for box-shadow. Can you remind me?